core/domain/html_validation_service_test.py

```diff
@@ -1270,20 +1270,46 @@ def test_convert_svg_diagram_tags_to_image_tags(self):
             'expected_output': (
                 '<oppia-noninteractive-image '
                 'alt-with-value="&amp;quot;Image&amp;quot;" '
-                'caption-with-value="" '
+                'caption-with-value="&amp;quot;&amp;quot;" '
                 'filepath-with-value="&amp;quot;img12.svg&amp;quot;">'
                 '</oppia-noninteractive-image>'
                 '<oppia-noninteractive-image '
                 'alt-with-value="&amp;quot;Image123&amp;quot;" '
-                'caption-with-value="" '
+                'caption-with-value="&amp;quot;&amp;quot;" '
                 'filepath-with-value="&amp;quot;img2.svg&amp;quot;">'
                 '</oppia-noninteractive-image>'
                 '<oppia-noninteractive-image '
                 'alt-with-value="&amp;quot;Image12345&amp;quot;" '
-                'caption-with-value="" '
+                'caption-with-value="&amp;quot;&amp;quot;" '
                 'filepath-with-value="&amp;quot;igage.svg&amp;quot;">'
                 '</oppia-noninteractive-image>'
             )
+        }]
+        for test_case in test_cases:
+            self.assertEqual(
+                html_validation_service.convert_svg_diagram_tags_to_image_tags(
+                    test_case['html_content']),
+                test_case['expected_output'])
+
+    def test_no_convertion_of_non_interactive_image_tags(self):
+        """Test that the convert_svg_diagram_tags_to_image_tags does not make
+         any changes in already existing oppia-noninteractive image tags.
```

Comment:
Nit: please remove the extra space before 'any' (extra lines should be aligned with the first ``"`` of the docstring).

Reply:
Done.